src/app.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+const { generateNumber } = require('./modules/generateNumber');
+const { makeTry } = require('./modules/makeTry');
+
+const winCombination = generateNumber();
```

Comment:
Would be better to rename it to the `secretNumber`
```suggestion
const secretNumber = generateNumber();
```